libraries/botbuilder-ai/tests/TestData/LanguageTranslator/should_bypass_calling_service_in_middleware_for_non-message_activities..json

```diff
@@ -0,0 +1 @@
+""
```

Comment:
This filename has two dots before extension

Reply:
That may be due to the Test name itself having a dot at the end of the text, and the file names were generated automatically. I'll correct that in both places, thanks.